go-controller/pkg/ovn/hybrid.go

```diff
@@ -25,36 +25,13 @@ import (
 // hybridOverlayNodeEnsureSubnet allocates a subnet and sets the
 // hybrid overlay subnet annotation. It returns any newly allocated subnet
 // or an error. If an error occurs, the newly allocated subnet will be released.
-func (oc *DefaultNetworkController) hybridOverlayNodeEnsureSubnet(node *kapi.Node, annotator kube.Annotator) (*net.IPNet, error) {
-	var existingSubnets []*net.IPNet
-
+func (oc *DefaultNetworkController) hybridOverlayNodeEnsureSubnet(node *kapi.Node) (*net.IPNet, error) {
```

Comment:
So now that the subnet allocation has been moved to the cluster manager, this `hybridOverlayNodeEnsureSubnet` only returns the `HybridOverlayNodeSubnet` annotation. Does this function still make sense or can we just use `houtil.ParseHybridOverlayHostSubnet`?

Reply:
Makes sense.  I'll delete this function